SD_DSC.adoc

```diff
@@ -456,22 +456,18 @@ The evaluator shall iterate through each of the methods selected by the ST and p
 The evaluator shall iterate through each of the methods selected by the ST and confirm that the KMD describes the applicable selected methods.
 
 ===== FCS_CKM.2 Cryptographic Key Distribution
-[IMPORTANT]
-====
-This has been changed from establishment to distribution
-====
 
 ====== TSS
 
-The evaluator shall examine the TSS to ensure that ST supports at least one key establishment scheme. The evaluator also ensures that for each key establishment scheme selected by the ST in FCS_CKM.2.1 it also supports one or more corresponding methods selected in FCS_COP.1/KAT. If the ST selects RSA in FCS_CKM.2.1, then the TOE must support one or more of "KAS1," or "KAS2," "KTS-OAEP," from FCS_COP.1/KAT. If the ST selects elliptic curve-based, then the TOE must support one or more of "ECDH-NIST" or "ECDH-BPC" from FCS_COP.1/KAT. If the ST selects Diffie-Hellman-based key establishment, then the TOE must support "DH" from FCS_COP.1/KAT.
+The evaluator shall examine the TSS to determine whether it describes the supported methods for key distribution functionality consistent with the claim(s) made in FCS_CKM.2.1. The evaluator shall confirm that the matching SFRs are included in the ST based on the claim(s) made in FCS_CKM.2.1.
```

Comment:
Maybe change claim(s) to selection(s)? Similar to what you write below under Test

Reply:
changed